src/hotspot/share/runtime/os.cpp

```diff
@@ -1487,7 +1487,9 @@ static const char* errno_to_string (int e, bool short_text) {
     X(ENETUNREACH, "Network unreachable") \
     X(ENFILE, "Too many files open in system") \
     X(ENOBUFS, "No buffer space available") \
+#if defined(__clang__)
```

Comment:
Interesting.  These are not clang specifics, these are errno values.  It would be better to do this in terms of the value itself.  E.g.

```
#ifdef ENODATA
  X(ENODATA, "...") \
#endif
```

Reply:
Ok

Reply:
forget this patch, it doesn't build. I can't put #ifdef inside a #define.